### ISSUE TITLE
Using plan "tiger" results in number of nodes error

### ISSUE THREAD

**Issue by tomer-ds** [cloudamqp/terraform-provider-cloudamqp#165]
Hi...

We have this configuration

```hcl
resource "cloudamqp_instance" "instance" {
  count = var.create_camqp_instance == true && var.create_camqp_vpc == false ? 1 : 0

  name        = local.full_environment_name
  plan        = var.camqp_instance_plan_name
  region      = join("::", ["amazon-web-services", var.aws_region])
  rmq_version = var.camqp_instance_version
}
```

And this configuration:
```hcl
var.create_camqp_instance = true
var.create_camqp_vpc = false
local.full_environment_name = "Some-Env"
var.camqp_instance_plan_name = "tiger"
var.aws_region = "eu-west-2"
var.camqp_instance_version =  "" # this takes the default version
```

I keep getting this error

```text
Error: CreateInstance failed, status: 400, message: map[errors:[map[nodes:Invalid number of nodes]]]
```

Expected behaviour:
- instance will be created on shared tiger plan

Actual result:
- error regarding number of nodes on plan where number of nodes is not a supplied value

Maybe something I misconfigured

**Comment by tbroden84** [comment]
@tomer-ds Yes is a misconfiguration, where nodes are provided to our backend. Should not be the case for shared plan. Will take a look at the issue.

**Comment by tbroden84** [comment]
@tomer-ds Added a fix to this and created a new release [v1.19.1](https://github.com/cloudamqp/terraform-provider-cloudamqp/releases/tag/v1.19.1).
